README.md

```diff
@@ -78,145 +135,201 @@ See [third_party_integration](./third_party_integration) for examples of code mo
 ## System requirements
 - Ubuntu\* 18.04 or later (64-bit)
 - Python\* 3.6.2 or later
-- NVidia CUDA\* Toolkit 10.2 or later^
-- PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)
+- Supported frameworks:
+  - PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)
+  - TensorFlow\* 2.4.2
 
-*NOTE:* The best known PyTorch version for the current NNCF is 1.8.1, and it is highly recommended to use it.
-
-^ If a torch package built for specific CUDA version is already present in the environment into which NNCF is being installed,
-and if it has a matching base version, then the CUDA version for which the present torch package is targeted will be used.
-
-Otherwise NNCF will install the latest available torch version from pip, which is targeted to the CUDA version of PyTorch packaging strategy's choosing.
-For PyTorch 1.8.1, the default CUDA is 10.2.
+This repository is tested on Python* 3.6.2+, PyTorch* 1.8.1 (NVidia CUDA\* Toolkit 10.2) and TensorFlow* 2.4.2 (NVidia CUDA\* Toolkit 11.0).
 
 ## Installation
 We suggest to install or use the package in the [Python virtual environment](https://docs.python.org/3/tutorial/venv.html).
 
+First, create a virtual environment with the version of Python you're going to use and activate it.
 
-#### As a package built from a checked-out repository:
-1) Install the following system dependencies:
-
-`sudo apt-get install python3-dev`
+Then you need to install one of the deep learning frameworks [PyTorch](https://pytorch.org/) or [TensorFlow](https://www.tensorflow.org/). Please follow to [TensorFlow installation guide](https://www.tensorflow.org/install/) or [PyTorch installation guide](https://pytorch.org/get-started/locally/#start-locally).
```

Comment:
I propose to rephrase it a little like "If you want to optimize a model from PyTorch, install ..."

Reply:
Done